scripts/install-systemd-multi-user.sh

```diff
@@ -97,6 +97,7 @@ poly_configure_nix_daemon_service() {
         task "Setting up the nix-daemon systemd service"
 
         _sudo "to create the nix-daemon tmpfiles config" \
+              mkdir -p "$(dirname "$TMPFILES_DEST")" \
               ln -sfn "/nix/var/nix/profiles/default$TMPFILES_SRC" "$TMPFILES_DEST"
```

Comment:
Sorry, I wasn't super clear. I think we need something likes:

```suggestion
        _sudo "to create parent of the nix-daemon tmpfiles config" \
              mkdir -p "$(dirname "$TMPFILES_DEST")"
              
        _sudo "to create the nix-daemon tmpfiles config" \
              ln -sfn "/nix/var/nix/profiles/default$TMPFILES_SRC" "$TMPFILES_DEST"
```